qelem/lib/screens/profile/EditProfile.dart

```diff
@@ -0,0 +1,117 @@
+                  height: 30,
+                  width: 30,
+                  decoration: BoxDecoration(
+                      color: const Color.fromARGB(255, 207, 165, 215),
+                      borderRadius: BorderRadius.circular(3),
+                      boxShadow: const [
+                        BoxShadow(
+                          color: Colors.black45,
+                          offset: Offset(0.0, 1.0),
+                          blurRadius: 2.0,
+                          spreadRadius: 0.1,
+                        )
+                      ]),
+                  child: IconButton(
+                    padding: const EdgeInsets.all(0.0),
+                    icon: const Icon(Icons.camera_alt),
+                    onPressed: () {},
+                  ),
+                ),
+              ],
+            ),
+          )),
+      Padding(
+        padding: const EdgeInsets.only(
+            top: 40.0, left: 15.0, right: 15.0, bottom: 15.0),
+        child: TextFormField(
+          style: textStyle,
+          controller: userNameController,
+          decoration: InputDecoration(
+              labelText: 'User Name',
+              labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+              border:
+                  OutlineInputBorder(borderRadius: BorderRadius.circular(5.0))),
+        ),
+      ),
+      Padding(
+        padding: const EdgeInsets.symmetric(vertical: 15.0, horizontal: 15.0),
+        child: TextFormField(
+          style: textStyle,
+          controller: firstNameController,
+          decoration: InputDecoration(
+              labelText: 'First Name',
+              labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+              border:
+                  OutlineInputBorder(borderRadius: BorderRadius.circular(5.0))),
+        ),
+      ),
+      Padding(
+        padding: const EdgeInsets.symmetric(vertical: 15.0, horizontal: 15.0),
+        child: TextFormField(
+          style: textStyle,
+          controller: lastNameController,
+          decoration: InputDecoration(
+              labelText: 'Last Name',
+              labelStyle: GoogleFonts.comfortaa(textStyle: textStyle),
+              border:
+                  OutlineInputBorder(borderRadius: BorderRadius.circular(5.0))),
+        ),
+      ),
+      Container(
+          margin: const EdgeInsets.only(right: 15.0, top: 15.0),
+          child: Align(
+            alignment: Alignment.centerRight,
+            child: SizedBox(
+              width: 115.0,
+              height: 45.0,
+              child: MaterialButton(
+                shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(5)),
+                child: Text(
+                  'UPDATE',
+                  style: GoogleFonts.comfortaa(
+                      color: Colors.white,
+                      textStyle: textStyle,
+                      fontSize: 16,
+                      letterSpacing: 2),
+                ),
```

Comment:
Just set it to the button variant of the primary text theme, It will allow us to define it in one place.